UserInterfaceLayer/src/main/java/com/axinalis/controller/UserInterfaceController.java

```diff
@@ -0,0 +1,66 @@
+package com.axinalis.controller;
+
+import com.axinalis.consumer.ClientsConsumer;
+import com.axinalis.consumer.TransactionsConsumer;
+import com.axinalis.model.Transaction;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.regex.Pattern;
+
+@RestController
+@RequestMapping("kafka")
+public class UserInterfaceController {
+
+    private KafkaTemplate<String, String> template;
+    private ClientsConsumer clientsConsumer;
+    private TransactionsConsumer transactionsConsumer;
+    @Value("${CREATE_USER_TOPIC}")
+    private String createUserTopic;
+    @Value("${TRANSACTIONS_TOPIC}")
+    private String transactionsTopic;
+    private ObjectMapper mapper;
+
+    public UserInterfaceController(@Autowired KafkaTemplate<String, String> template,
+                                   @Autowired ClientsConsumer clientsConsumer,
```

Comment:
to simplify, may propose you to create dedicated KafkaService that will handle all Kafka communications.